src/SFA.DAS.EmployerIncentives.Functions.LegalEntities/Services/LegalEntities/ILegalEntitiesService.cs

```diff
@@ -9,5 +9,8 @@ public interface ILegalEntitiesService
         Task Refresh(int pageNumber, int pageSize);
         Task Add(AddRequest request);
         Task Remove(RemoveRequest request);
+        Task UpdateVrfCaseDetails();
```

Comment:
Should they be in their own interface to do with VRF cases rather than legal entities (Interface Segregation Principle)
?

Reply:
👍 That's done.